### ISSUE TITLE
[BUG] Model Estimation and Model Tuning modules seem similar

### ISSUE THREAD

**Issue by paulf81** [NREL/flasc#144]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Model estimation and model tuning modules in FLASC have different functionality, but are notionally related, should probably be harmonized into one module in the future

### Environment

```markdown
- OS:
- pip environment (can be retrieved with `pip list`):
```

**Comment by Bartdoekemeijer** [comment]
Sounds like a great idea!

**Comment by misi9170** [comment]
Addressed in #165
